contracts/migration/baobab/Aggregator/migration.lock

```diff
@@ -55,3 +55,8 @@
 20230925130994_UNI-USDT_register.json
 20230925130994_XRP-USDT_register.json
 20231022211071_JOY-USDT.json
+20231102141119_BORA-KRW.json
+20231102141119_KSP-KRW.json
+20231102141119_MBX-KRW.json
+20231102141119_WEMIX-USDT.json
+accountList.json
```

Comment:
@nick-bisonai Please remove this `accountList.json`

Reply:
removed it! thank you for pointing out